src/index.html

```diff
@@ -21,6 +21,8 @@
     <meta property="og:description" content="EvalAI is an open-source web
       platform for organizing and participating in challenges to push the state
       of the art on AI tasks." />
+    <link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.0/cookieconsent.min.css" />
+    <script src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.1.0/cookieconsent.min.js"></script>
```

Comment:
This looks JS. Can we do using angular dependency ? Also can you make sure it is not increasing the loading time of the site?